### PR TITLE
Rename get_binstar

### PR DESCRIPTION
**Pull request by srossross**
`get_binstar` is now deprecated in favor of `get_server_api`
`get_server_api` also has a different function signature, It takes individual arguments rather than a single `args` object.

### CODE REVIEW

**Comment by dsludwig** [comment]
@srossross this has bit-rotted, can you update it?

**Comment by srossross** [comment]
Assigning this PR 0.5 story points because it is an old PR without a parent issue.

**Comment by rafaelhbarros** [comment]
LG trade mark

File: binstar_client/utils/__init__.py [hunk 1]
```diff
@@ -149,30 +150,56 @@ def load_token(url):
...
+    """
+
+    warnings.warn(
+      'method get_binstar is depricated, please use `get_server_api`',
+      DeprecationWarning
+    )
+
+    token = getattr(args, 'token', None)
+    log_level = getattr(args, 'log_level', 0)
+    site = getattr(args, 'site', 0)
```
Comment (dsludwig) [inline binstar_client/utils/__init__.py]: is `0` the correct default for `site`? In `get_server_api`, the default is `None`

Comment (srossross) [inline binstar_client/utils/__init__.py]: Correct

**Comment by dsludwig** [comment]
lgtm

**Pull request merged**